airbyte-integrations/connectors/destination-gcs/src/test-integration/java/io/airbyte/integrations/destination/gcs/GcsAvroParquetDestinationAcceptanceTest.java

```diff
@@ -85,7 +91,7 @@ private JsonNode getJsonNode(final AirbyteStream stream, final String name) {
   }
 
   private Set<Type> getExpectedSchemaType(final JsonNode fieldDefinition) {
-    final JsonNode typeProperty = fieldDefinition.get("type");
+    final JsonNode typeProperty = fieldDefinition.get("type") == null ? fieldDefinition.get("$ref") : fieldDefinition.get("type");
```

Comment:
Nice work, suggestion to maybe considering a comment here that notes that `$ref` is utilized as a result of the version migration and that `type` is the original standard since tests are a great way to maintain knowledge of business context

Reply:
The `type` is still valid since we have object and arrays (for example: `"type": ["object"]`) in schemas but it will be worth to mentioned the `$ref` in the comments since it's not obvious. Thanks for suggesting!